keyboards/keebio/levinson/keymaps/omgvee/keymap.c

```diff
@@ -0,0 +1,205 @@
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,     MS_L,MS_D,MS_U,MS_R,WH_L,WH_R,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,     BTN1,WH_D,WH_U,BTN2,    ,    ,
+  //|----+----+----+----+----+----|    |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,         ,    ,    ,    ,    ,
+  //`----+----+----+----+----+----'    `----+----+----+----+----+----'
+  ),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset|RGB Tg|RGB Md|Hue Up|Hue Dn|Sat Up|Sat Dn|Val Up|Val Dn|      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Qwerty|Colemk|Dvorak|      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+  [_ADJUST] = LAYOUT( \
+      RESET,  QWERTY, COLEMAK,  DVORAK, _______, EEP_RST,   KC_MNXT, KC_MPRV, KC_MFFD, KC_MRWD,  DVORAK, KC_EJCT, \
+    RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, KC_PGUP, KC_HOME,   KC_LEFT, KC_DOWN,   KC_UP, KC_RGHT,   CK_UP, CK_DOWN, \
+    RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, KC_PGDN,  KC_END,   KC_BRID, KC_BRIU, KC_VOLD, KC_VOLU, KC_MUTE, KC_MPLY, \
+    BL_TOGG,  BL_INC,  BL_DEC, BL_BRTG, _______, _______,    CK_RST, _______, CK_TOGG,  MU_MOD,  MU_TOG,  AU_TOG \
+  )
+
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+float tone_dvorak[][2]     = SONG(DVORAK_SOUND);
+float tone_colemak[][2]    = SONG(COLEMAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_qwerty);
+        #endif
+        persistent_default_layer_set(1UL<<_QWERTY);
+      }
+      return false;
+      break;
+    case COLEMAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_colemak);
+        #endif
+        persistent_default_layer_set(1UL<<_COLEMAK);
+      }
+      return false;
+      break;
+    case DVORAK:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_dvorak);
+        #endif
+        persistent_default_layer_set(1UL<<_DVORAK);
```

Comment:
```suggestion
        set_single_persistent_default_layer(_DVORAK);
```

Reply:
aye, I keep forgetting. I've 'inherited' bakingpy's keymap, so I forgot to check for these deprecated things. Sorry about that and misusing your time for minor crap :) I'll try to be more careful with my next keymaps and whatnot.

Btw, shouldn't we let bakingpy know to fix his keymaps as well? Or maybe I'll send him a pull request as well, save some time. 

thank you btw, for taking the time to check my stuff, I appreciate it :)